variables.tf

```diff
@@ -223,7 +223,7 @@ variable "transit_gateway_connections" {
 ##############################################################################
 
 variable "ssh_keys" {
-  description = "SSH Keys to use for VSI Provision. Must be RSA keys with a key size of either 2048 bits or 4096 bits (recommended). If `public_key` is not provided, the named key will be looked up from data. If a resource group name is added, it must be included in `var.resource_groups`"
+  description = "SSH keys to use to provision a VSI. Must be an RSA key with a key size of either 2048 bits or 4096 bits (recommended). If `public_key` is not provided, the named key will be looked up from data. If a resource group name is added, it must be included in `var.resource_groups`. See https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys."
```

Comment:
We should also update the ssh_key variable under the examples directory

Reply:
Thought about that but saw that it was already different, so left it. I'll put it in the examples.